app/models/wpcc/period_mapper.rb

```diff
@@ -1,34 +1,22 @@
 module Wpcc
-  class PeriodFilter
+  class PeriodMapper
```

Comment:
Not sure about the name of this class. My first thought was mapping to what? Would PeriodPercentsMapper be more explicit?

Reply:
I had he same doubt then went for simplicity, but I like `PeriodPercentsMapper` 👍